src/algebra/group/units_hom.lean

```diff
@@ -30,6 +30,4 @@ instance : is_monoid_hom (coe : units α → α) :=
 
 lemma map_comp : map (g ∘ f) = map g ∘ map f := rfl
```

Comment:
Even though I theoretically prefer the version that you kept,
 the other version
```lean
lemma map_comp' : map (λ x, g (f x)) = λ x, map g (map f x) := rfl
```
seems to be more useful in practice. Otherwise people first have to fold their composition into some thing that uses `\circ`.

Reply:
I think that `@[inline,reducible]` on `function.comp` makes these two forms equivalent. I just tried to prove `map_comp'` using `by rw map_comp`, and it worked.